Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -280,6 +281,11 @@ public override string GetDriverConfigsEditPageName(Agent.eDriverType driverSubT
         [UserConfiguredDescription("Specifies the state of current session’s user prompt handler, You can change it from dismiss, accept, dismissAndNotify, acceptAndNotify, ignore")]
         public string UnhandledPromptBehavior { get; set; }
 
+        [UserConfigured]
+        [UserConfiguredDefault("")]
+        [UserConfiguredDescription("Specify domains to be blocked for browser session")]
+        public string BlockedDomains { get; set; }
```

Comment:
Please specify compatible browsers in description

Reply:
Done. Removed the property since its not needed.